staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -74,15 +76,11 @@ type AuditWebhookOptions struct {
 	ConfigFile string
 	// Should the webhook asynchronous batch events to the webhook backend or
 	// should the webhook block responses?
-	//
-	// Defaults to asynchronous batch events.
-	Mode string
 }
 
 func NewAuditOptions() *AuditOptions {
 	return &AuditOptions{
-		WebhookOptions: AuditWebhookOptions{Mode: pluginwebhook.ModeBatch},
```

Comment:
Why remove WebhookOptions from the initialization?

Reply:
I thought Log and webhook should use same mode. But now understand both options should reserve.

Reply:
I thought log and webhook should use same mode: both blocking or buffered.   Now it is explicit.